vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -61,6 +61,12 @@ instances:
     #
     # ssl_capath: "<DIRECTORY_PATH>"
 
+    ## @param tls_ignore_warning - boolean - optional - default: false
+    ## If `tls_verify` is disabled, security warnings are logged by the check when making http requests.
```

Comment:
Just curious- if `tls_verify` is not disabled, would this ever have an effect?

Reply:
Nope, it doesn't have an effect.

Reply:
Ok, I see. Looks good!